web_src/js/features/tablesort.js

```diff
@@ -0,0 +1,20 @@
+export default function initTableSort() {
+  for (const header of document.querySelectorAll('th[data-sortt-asc]') || []) {
+    const {sorttAsc, sorttDesc, sorttDefault} = header.dataset;
+    header.addEventListener('onclick', () => {
```

Comment:
```suggestion
    header.addEventListener('click', () => {
```

Should fix it, sorry.